core/src/instructions/close_account.rs

```diff
@@ -0,0 +1,47 @@
+use generic_array_struct::generic_array_struct;
+
+use crate::instructions::internal_utils::impl_memset;
+
+// Accounts
+
+#[generic_array_struct(builder pub)]
+#[derive(Debug, Default, Clone, Copy, PartialEq, Eq, Hash)]
+#[repr(transparent)]
+pub struct CloseAccountIxAccs<T> {
+    pub account_to_close: T,
+    pub refund_rent_to: T,
```

Comment:
nit: i think lets try to keep names short where possible.

Also normally i would be fine with `refund_rent_to` since it very clearly describes the purpose of the account, but in the case of wsol, its not just rent thats refunded so probably `dst` is more accurate, and more in-line with [upstream naming conventions](https://docs.rs/spl-token-2022/latest/spl_token_2022/instruction/enum.TokenInstruction.html#variant.CloseAccount)  
 
```suggestion
    pub close: T,
    pub dst: T,
```